modules/light-clients/08-wasm/types/msgs.go

```diff
@@ -47,3 +50,46 @@ func (m MsgPushNewWasmCode) GetSigners() []sdk.AccAddress {
 func (msg MsgPushNewWasmCode) GetSignBytes() []byte {
 	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&msg))
 }
+
+// NewMsgUpdateWasmCodeId creates a new MsgUpdateWasmCodeId instance
+//
+//nolint:interfacer
+
+// Route Implements Msg.
+func (msg MsgUpdateWasmCodeId) Route() string { return ModuleName }
+
+// Type Implements Msg.
+func (msg MsgUpdateWasmCodeId) Type() string { return TypeMsgUpdateWasmCodeId }
+
+// TODO: add tests for MsgUpdateWasmCodeId
```

Comment:
Can you add test cases for `MsgUpdateWasmCodeId` in this PR? If this is an urgent to be merged without having test cases, then i would suggest to create an issue to address the TODO comment separately at a later time.

Reply:
Added

Reply:
Can you also remove TODO comment? @vmarkushin

Reply:
done